pageserver/src/page_service.rs

```diff
@@ -1275,6 +1279,35 @@ where
                 ctx,
             )
             .await?;
+        } else if let Some(params) = parts.strip_prefix(&["pagestream_v3"]) {
```

Comment:
nit: i would move this at the top of the if-else chain. it's nice to have the default at the top.

Reply:
Moved